jobs/integr8ly/ocp3/osd/create-rhmi-user.yaml

```diff
@@ -0,0 +1,123 @@
+          default: 'master'
+          description: 'Branch of the repository'
+      - string:
+          name: CLUSTER_URL
+          description: 'URL of cluster on which the smoke tests will be executed.'
+      - string:
+          name: ADMIN_USERNAME
+          default: 'admin@example.com'
+          description: 'Admin sername to login to Integreatly cluster.'
+      - string:
+          name: ADMIN_PASSWORD
+          default: 'Password1'
+          description: 'Admin password to login to Integreatly cluster.'
+      - string:
+          name: CUSTOMER_ADMIN_USERNAME
+          default: 'customer-admin@example.com'
+          description: 'customer-admin username to login to Integreatly cluster.'
+      - string:
+          name: CUSTOMER_ADMIN_PASSWORD
+          default: 'Password1'
+          description: 'Password to login to Integreatly cluster.'
+      - string:
+          name: NAMESPACE_PREFIX
+          description: "Value used to prefix the names of the namespaces created during Integr8ly installation"
+      - string:
+          name: WEBAPP_URL
+          description: 'URL of Tutorial Web App'
+      - bool:
+          name: CLEAN_RESOURCES
+          default: true
+          description: 'depending on whether the resources should be cleaned after the successful execution'
+      - string:
+          name: SSO_URL
+          description: 'SSO URL for the cluster on which the tests will be executed (sso-user-create test)'
+      - string:
+          name: USERNAME
+          default: "osdtester"
+          description: 'Name of newly created user. "01" will be appended.'
+      - string:
+          name: TIMEOUT_THRESHOLD
+          default: '1'
+          description: 'optionally increase timeout values. If you provide value 2 it means it will be sleeping/waiting two times longer'      
+    dsl: |
+        jobParams = [
+            string(name: 'REPOSITORY', value: "${REPOSITORY}"),
+            string(name: 'BRANCH', value: "${BRANCH}"),
+            string(name: 'CLUSTER_URL', value: "${CLUSTER_URL}"),
+            string(name: 'ADMIN_USERNAME', value: "${ADMIN_USERNAME}"),
+            string(name: 'ADMIN_PASSWORD', value: "${ADMIN_PASSWORD}"),
+            string(name: 'CUSTOMER_ADMIN_USERNAME', value: "${CUSTOMER_ADMIN_USERNAME}"),
+            string(name: 'CUSTOMER_ADMIN_PASSWORD', value: "${CUSTOMER_ADMIN_PASSWORD}"),
+            string(name: 'NAMESPACE_PREFIX', value: "${NAMESPACE_PREFIX}"),
+            string(name: 'WEBAPP_URL', value: "${WEBAPP_URL}"),
+            booleanParam(name: 'CLEAN_RESOURCES', value: Boolean.valueOf("${CLEAN_RESOURCES}")),
+            string(name: 'SSO_URL', value: "${SSO_URL}"),
+            string(name: 'USERNAME', value: "${USERNAME}"),
+            string(name: 'TIMEOUT_THRESHOLD', value: "${TIMEOUT_THRESHOLD}"),
+            string(name: 'NUMBER_OF_USERS', value: '1'),
+        ]
+
+        try {
+            timeout(180) { ansiColor('gnome-terminal') { timestamps {
```

Comment:
timeout value here is probably very generous (probably not a big deal though )

Reply:
it is, will decrease to 40 minutes.